modules/manually-create-identity-access-management.adoc

```diff
@@ -221,8 +221,72 @@ $ rm 0000_30_capi-operator_00_credentials-request.yaml
 ----
 endif::cco-manual-mode[]
 
+. Copy the generated credential files to the target manifests directory:
++
+[source,terminal]
+----
+$ cp ./<path_to_ccoctl_output_dir>/manifests/* ./<path_to_installation_dir>/manifests/
+----
++
+where:
+
+ifdef::alibaba-default,alibaba-custom[]
+`<path_to_ccoctl_output_dir>`:: Specifies the directory created by the `ccoctl alibabacloud create-ram-users` command.
+endif::alibaba-default,alibaba-custom[]
+`<path_to_installation_dir>`:: Specifies the directory in which the installation program creates files.
+
 . Create YAML files for secrets in the `openshift-install` manifests directory that you generated previously. The secrets must be stored using the namespace and secret name defined in the `spec.secretRef` for each `CredentialsRequest` object. The format for the secret data varies for each cloud provider.
 
+.. Create an IAM user for each cluster component.
+.. Generate and attach a policy to the newly created IAM user with the permissions from the credentials request manifest generated in the previous step.
+.. Generate a secret access key for the newly created IAM user.
+.. Create a kubernetes secret YAML file with the secret access key and add it to the installer manifests directory:
++
+[source,yaml]
+ifdef::aws[]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: openshift-cloud-credential-operator <1>
+  name: cloud-credential-operator-iam-ro-creds <2>
+data:
+  aws_access_key_id: Base64encodeAccessKeyID
```

Comment:
@akhil-rane are the various `Base64encode*` values meant to be changed by the user? I assume so, but just want to confirm